compiler/test/test_end_to_end.re

```diff
@@ -1036,14 +1036,14 @@ let loop_tests = [
 
 let oom = [
   tgcerr("oomgc1", 70, "(1, (3, 4))", "Maximum memory size exceeded"),
-  tgc("oomgc2", 356, "(1, (3, 4))", "(1, (3, 4))"),
-  tgc("oomgc3", 256, "(3, 4)", "(3, 4)"),
+  tgc("oomgc2", 880, "(1, (3, 4))", "(1, (3, 4))"),
+  tgc("oomgc3", 440, "(3, 4)", "(3, 4)"),
```

Comment:
These went up because the `toString` algorithm changed a bit. Thoughts to improve in the description.

As a side note, I think the GC tests would be better served if we didn't have the result printed—we could focus on the memory usage of the exact program written.

Reply:
Needed to fix the GC tests to work with now the CLI requires `toString` independently.